### PR TITLE
docs: improve cluster documentation

### PR DESCRIPTION
**Pull request by nqb**
# Description
- Refactor cluster documentation to simplify installation
- Reuse common sections across documentation (maintenance patches) in place of links between documents
- Make troubleshooting section more clear

# Impacts
Cluster installation.

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
I will have to set up a cluster soon so I will refresh this PR.

**Comment by nqb** [comment]
Ready for review. I will test it soon because I'm unsure about behavior related to deactivation of `galera-autofix` service during installation.

Also, section `5.3. Recovering when a node is missing` is unclear for me.

File: docs/common/enable_galera_autofix.asciidoc [hunk 1]
```diff
@@ -0,0 +1 @@
+==== Enable galer
```
Comment (extrafu) [inline docs/common/enable_galera_autofix.asciidoc]: broken title

Comment (nqb) [reply docs/common/enable_galera_autofix.asciidoc]: removed.

**Comment by lzammit** [comment]
Beside my comment.

image

**Pull request merged**